environments/nodejs/builder/Dockerfile-12.16

```diff
@@ -0,0 +1,11 @@
+ARG BUILDER_IMAGE=fission/builder
+FROM ${BUILDER_IMAGE}
+# default variant is the official alpine node image (much smaller than the standard image)
+FROM node:12.16-alpine3.11
```

Comment:
The latest LTS on website seems to be 12.18.3 I am not sure how big the difference is, can you please confirm if we are ok with 12.16 or should bump to higher versions?